plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/p3order/LayerSweepCrossingMinimizer.java

```diff
@@ -229,7 +229,8 @@ private int minimizeCrossingsWithCounter(final GraphInfoHolder gData) {
         
         if ((!gData.lGraph().getProperty(InternalProperties.FIRST_TRY_WITH_INITIAL_ORDER)
                 && !gData.lGraph().getProperty(InternalProperties.SECOND_TRY_WITH_INITIAL_ORDER))
-                || gData.lGraph().getProperty(LayeredOptions.CONSIDER_MODEL_ORDER_STRATEGY) == OrderingStrategy.NONE) {
+                || gData.lGraph().getProperty(LayeredOptions.CONSIDER_MODEL_ORDER_STRATEGY) == OrderingStrategy.NONE
+                || gData.lGraph().getProperty(LayeredOptions.INTERACTIVE_LAYOUT) /* FIXME */) {
```

Comment:
why is there a FIXME?

Reply:
You added `gData.lGraph().getProperty(LayeredOptions.INTERACTIVE_LAYOUT)`. This should be removed and model order should be deactivated by the `LayeredInteractiveGraühVisitor`